spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Bilinear.scala

```diff
@@ -160,7 +160,7 @@ class Bilinear[T: ClassTag](
     gradInput
   }
 
-  override def accGradParameters(input: Table, gradOutput: Tensor[T], scale: Double = 1.0): Unit = {
+  override def accGradParameters(input: Table, gradOutput: Tensor[T]): Unit = {
```

Comment:
please check if scaleW and scaleB == 0, if so, just return or by pass the computing